test/testutilsio.cpp

```diff
@@ -77,6 +77,13 @@ private slots:
             QCOMPARE(readResult.value(), size10Data);
         }
 
+        {
+            const Path crlfFile = testFolder / Path(u"crlf.txt"_s);
+            const auto readResult = Utils::IO::readFile(crlfFile, -1, QIODevice::Text);
+            QCOMPARE(readResult.has_value(), true);
+            QCOMPARE(readResult.value(), "\n\n");
+        }
```

Comment:
Will it still test this condition  if previous test failed?

Reply:
Not sure what you mean. The tests will stop immediately when encounter error.

Reply:
> Not sure what you mean. The tests will stop immediately when encounter error.

Sorry. Due to the inconvenience of the GitHub mobile web interface, this comment turned out to be near the wrong line of code. I asked exactly what you answered.